build_external/README.md

```diff
@@ -22,7 +22,7 @@ For configurations that involve building and running the agent alone, we still u
 `docker-compose` for consistency with more complex configurations. The more complex
 configurations allow the agent to be run in conjunction with parts of the cloud
 infrastructure (these parts of the code are not public), or with external brokers
-(such as VerneMQ for MQTT), or with other external tools (such as TSDB to allow the agent to
+(such as VerneMQ for MQ Telemetry Transport (MQTT)), or with other external tools (such as Time Series Database (TSDB) to allow the agent to
```

Comment:
**Time series database** isn't a formal name, and so should be written in lowercase.

```suggestion
(such as VerneMQ for MQ Telemetry Transport (MQTT)), or with other external tools (such as a time series database (TSDB) to allow the agent to
```